README.md

```diff
@@ -196,76 +196,33 @@ import com.rudderstack.generated.*
 
 ### RudderStack JavaScript SDK
 
-- Import the RudderTyper-generated client using `require()` and make the calls if your framework supports them. Otherwise, you can use [**Browserify**](https://browserify.org/) to generate a bundle that supports your implementation. The implementation for each of the alternatives mentioned above will be as shown:
+There are two ways to get started with Rudder-Typer in your browser:
 
-#### Using the `require()`method
+#### 1. Using CDN (Snippet)
 
-```javascript
-// Import RudderStack JS SDK and initialize it
-const RudderAnalytics = require('@rudderstack/analytics-js');
-
-const rudderAnalytics = new RudderAnalytics();
-rudderAnalytics.load(WRITE_KEY, DATA_PLANE_URL, {});
+- Paste the snippet from Rudderstack dashboard to your HTML file. [Ref](https://www.rudderstack.com/docs/sources/event-streams/sdks/rudderstack-javascript-sdk/quickstart/#using-cdn)
+- If you are using typescript add `@rudderstack/analytics-js` as dev dependency.
 
-// Import your auto-generated RudderTyper client:
-const rudderTyper = require('./analytics/index');
+```sh
+npm install --save-dev  @rudderstack/analytics-js
+```
 
-// Pass in your @rudderstack/analytics-js instance to RudderTyper client
-rudderTyper.setRudderTyperOptions({
-  analytics: rudderAnalytics,
-});
+- Execute the following command to generate a bundle from the RudderTyper client:
 
-// Issue your first RudderTyper track call!
-rudderTyper.orderCompleted({
-  orderID: 'ck-f306fe0e-cc21-445a-9caa-08245a9aa52c',
-  total: 39.99,
-});
+```sh
+npx browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
 ```
 
-#### Using `browserify`
-
-- Execute the following command to generate a bundle from the RudderTyper client:
+- For typescript project add `tsify` npm package as dev dependency and run the following command:
 
 ```sh
-browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
+npx browserify analytics/index.js -p [ tsify ] --standalone rudderTyper >  rudderTyperBundle.js
```

Comment:
This is the main change in the command from the previous instruction. Please be a little more cautious.
```suggestion
npx browserify analytics/index.ts -p [ tsify ] --standalone rudderTyper >  rudderTyperBundle.js
```